app/controllers/recipes_controller.rb

```diff
@@ -28,6 +29,11 @@ def destroy
     redirect_to recipes_url, alert: 'Recipe was successfully destroyed.'
   end
 
+  def list_public_recipes
```

Comment:
Would it make sense to rename this to `public_recipes`? This would keep consistency between the route's name and the controller's name.